src/Core/Services/AuthService.cs

```diff
@@ -268,12 +268,9 @@ private async Task<SymmetricCryptoKey> MakePreloginKeyAsync(string masterPasswor
                     kdfIterations = preloginResponse.KdfIterations;
                 }
             }
-            catch (ApiException e)
+            catch (ApiException e) when (e.Error == null || e.Error.StatusCode != System.Net.HttpStatusCode.NotFound)
             {
-                if (e.Error == null || e.Error.StatusCode != System.Net.HttpStatusCode.NotFound)
-                {
-                    throw e;
-                }
+                throw;
```

Comment:
I'm not sure why we catch this error just to rethrow it. I left it in there (removing the rethrow of course), but i'll remove it if anyone can confirm that its unnecessary.